raiden/raiden_service.py

```diff
@@ -452,6 +453,15 @@ def start_neighbours_healthcheck(self, chain_state):
     def get_block_number(self):
         return views.block_number(self.wal.state_manager.current_state)
 
+    def pop_payment_status(
```

Comment:
This function is not necessary at all, the lock is around an atomic operation `pop` (the first dictionary lookup always succeeds because it's a `defaultdict`